redaxo/src/addons/mediapool/lang/de_de.lang

```diff
@@ -104,3 +104,5 @@ var_medialist_move_top = Ausgewähltes Medium an den Anfang verschieben
 var_medialist_move_up = Ausgewähltes Medium nach oben verschieben
 var_medialist_move_down = Ausgewähltes Medium nach unten verschieben
 var_medialist_move_bottom = Ausgewähltes Medium an das Ende verschieben
+
+pool_file_upload_decline = Der Upload wurde durch Addon {0} verboten: {1}
```

Comment:
Die Meldung finde ich so nicht sinnvoll, die Redakteure brauchen den technischen Hintergrund nicht, welches Addons der Auslöser ist.
Ich würde durch den EP direkt die Message setzen lassen als Subject.

Reply:
Ich hatte den Namen mit reingenommen für Situationen, in denen der EP von mehreren Addons genutzt wird. Dann könnte es durchaus nützlich sein den "Übeltäter" zu erfahren wenn man nicht damit gerechnet hat, dass ein Upload abgelehnt wird.

Ein beispielhaftes Szenario (wie sinnvoll das auch immer sein mag): man hat zwei Addons die Bilder analysieren und u.a. anbieten, Bilder unterhalb einer bestimmten Auflösung abzulehnen. Allerdings wurde vergessen, den entsprechenden Wert für eines der Addons richtig anzupassen, sodass dieses nun Uploads ablehnt. Wenn das auch so kommuniziert ist, lassen sich solche Fehler schnell beheben.

Ich verstehe aber, dass die Info für normale Redakteure, die vielleicht gar nichts von dem Addon wissen auch verwirrend sein kann. **Ich könnte auch einfach zwei Meldungen erstellen: falls der User ein Admin ist, wird der Addon-Name mit angezeigt, ansonsten gibt es nur die Meldung.**

Reply:
Ich denke wir sollten die eigentliche meldung von der extension returnen lassen.

Dann die extension selbst den genauen grund returnen.

D.h. Von core seite den EP aufrufen. Wenn ein steing returned wird diesen als Fehlermeldung interpretieren

Reply:
Gut, dann setze ich das erstmal so um.